proto/evmos/incentives/v1/genesis.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
```

Comment:
These proto files were removed. We removed the proposals from the store on v16 upgrade, so we can safely remove these